orientation/exercises/04_TUPLES.md

```diff
@@ -19,6 +19,7 @@ subl .
 
 1. Create a tuple named `zoo` that contains your favorite animals.
 1. Find one of your animals using the `.index(value)` method on the tuple.
+<!-- TODO: Confirm this is what we want to do here. Should this be `foo in tuple` instead of a for loop? -->
```

Comment:
I'm fine with changing this to value lookup syntax

Reply:
👍